mix.exs

```diff
@@ -43,29 +46,28 @@ defmodule ImgProxy.MixProject do
 
   def package do
     [
+      description: "imgproxy URL generator and helper functions",
       files: ["lib", "mix.exs", "README*", "LICENSE*"],
       maintainers: ["Brian Muller"],
       licenses: ["MIT"],
       links: %{
-        "GitHub" => "https://github.com/bmuller/imgproxy",
-        "Changelog" => "https://github.com/bmuller/imgproxy/blob/master/CHANGELOG.md",
+        "Changelog" => "https://hexdocs.pm/imgproxy/changelog.html",
+        "GitHub" => @source_url,
         "imgproxy Site" => "https://imgproxy.net"
       }
     ]
   end
 
-  # Run "mix help compile.app" to learn about applications.
   def application do
     [
       extra_applications: [:logger]
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:credo, "~> 1.5", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.25", only: :dev}
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false}
```

Comment:
Let's go with a recent ex_doc requirement (at least 0.24, if not 0.25). I don't want to promise users that version 0.5.1 from 2014 will work, for instance - I have no idea if that's true.